WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -102,7 +103,7 @@ struct CartView: View {
 
 private extension CartView {
     enum Constants {
-        static let primaryFont: Font = .system(size: 40, weight: .bold, design: .default)
+        static let primaryFont: Font = Font.posTitle
```

Comment:
Bold = emphasized, it's set to `postTitleEmphasized` on trunk, so you can just merge/rebase the branch with trunk 👍 

```suggestion
        static let primaryFont: Font = Font.posTitleEmphasized
```

Reply:
Gotcha! Done on https://github.com/woocommerce/woocommerce-ios/pull/13631/commits/59ff42d3a14e58d767bec3cfc5b8dc4fd7024386